project/src/main/java/com/google/sps/data/EventVolunteering.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+
+/** Class that stores an event and an opportunityName. */
+public final class EventVolunteering {
+  private final Event event;
+  private final String opportunityName;
```

Comment:
Should we not let these be final? What will happen if the host edits event?

Reply:
This is a object that resembles a JAVA pair that holds the event and opportunity name. If the host edits the events the the database query will return the correct result because the database function creates a new EventVolunteering object with the current event data.